internal/provider/resource_wlan.go

```diff
@@ -137,6 +137,12 @@ func resourceWLAN() *schema.Resource {
 				Optional:    true,
 				Default:     true,
 			},
+			"uapsd": {
+				Description: "Enable Unscheduled Automatic Power Save Delivery",
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
```

Comment:
Does this default to false in the controller currently? I haven't checked it.

Reply:
Yes. If you use the UI and select create and expand the "advanced options", you will see this:
![Screenshot 2021-07-23 at 10-53-29 UniFi Network](https://user-images.githubusercontent.com/2661907/126719023-7aeba3cb-8f82-45b0-9e33-032d6dfba129.png)